e2e/testcases/login/5-validate-langiage-drop-down.spec.ts

```diff
@@ -0,0 +1,207 @@
+import { test, expect, type Page } from '@playwright/test'
+import { LOGIN_URL } from '../../constants'
+import { ensureLoginPageReady } from '../../helpers'
+
+let page: Page
+
+test.describe.configure({ mode: 'serial' })
```

Comment:
In this case, the serial mode is be a bit of an overkill and we should use shortcut for login and other processes instead

Reply:
Or just duplicate code at first per test. No harm in that and we can always then abstract if needed :)